x-pack/plugins/observability/public/embeddable/slo/slo_overview.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useEffect } from 'react';
+import { EuiIcon } from '@elastic/eui';
+import { Chart, Metric, MetricTrendShape, Settings } from '@elastic/charts';
+import numeral from '@elastic/numeral';
+import { ALL_VALUE } from '@kbn/slo-schema';
+import { EuiLoadingChart } from '@elastic/eui';
+import { euiStyled } from '@kbn/kibana-react-plugin/common';
+import { NOT_AVAILABLE_LABEL } from '../../../common/i18n';
+import { useKibana } from '../../utils/kibana_react';
+import { useFetchSloDetails } from '../../hooks/slo/use_fetch_slo_details';
+import { paths } from '../../../common/locators/paths';
+
+import { EmbeddableSloProps } from './types';
+
+export function SloOverview({ sloId, sloInstanceId, startTime, endTime }: EmbeddableSloProps) {
+  const {
+    uiSettings,
+    application: { navigateToUrl },
+    http: { basePath },
+  } = useKibana().services;
+  const { isLoading, slo, refetch, isRefetching } = useFetchSloDetails({
+    sloId,
+    instanceId: sloInstanceId,
+    shouldRefetch: true,
+  });
+
+  useEffect(() => {
+    refetch();
+  }, [startTime, endTime, refetch]);
+
+  const percentFormat = uiSettings.get('format:percent:defaultPattern');
+  const isSloNotFound = !isLoading && slo === undefined;
+
+  const getIcon = useCallback(
+    (type: string) =>
+      ({ width = 20, height = 20, color }: { width: number; height: number; color: string }) => {
+        return <EuiIcon type={type} width={width} height={height} fill={color} />;
+      },
+    []
+  );
+
+  const sloSummary = slo?.summary;
+  const sloStatus = sloSummary?.status;
+  let color;
+  switch (sloStatus) {
+    case 'HEALTHY':
+      color = '#e6f9f7';
```

Comment:
I tried to use `useEuiBackgroundColor('success');`, but I got:

`React Hook "useEuiBackgroundColor" is called conditionally. React Hooks must be called in the exact same order in every component render.`

That's why I used the hex colors.

Reply:
Ok your suggestion works, but there is no `euiTheme.colors.subdued` for the NO_DATA scenario. Mixing hex with euiTheme colors looks pretty weird, no?

```
let color;
  switch (sloStatus) {
    case 'HEALTHY':
      color = euiTheme.colors.success;
      break;
    case 'NO_DATA':
      color = '#f7f8fc';
      break;
    case 'DEGRADING':
      color = euiTheme.colors.warning;
      break;
    case 'VIOLATED':
      color = euiTheme.colors.danger;
      break;
    default:
      color = '#f7f8fc';
  }


```

Reply:
I will use the hex colors, cause the ones from euiTheme look different than the design
<img width="1618" alt="Screenshot 2023-09-22 at 21 09 12" src="https://github.com/elastic/kibana/assets/2852703/1b60f579-c0fc-4051-95ad-b606f4c8651d">

Reply:
We need to ask @maciejforcone what's the euiTheme colors we need to use

Reply:
Maybe one of these? https://eui.elastic.co/#/theming/colors/utilities#background-colors

Reply:
>React Hook "useEuiBackgroundColor" is called conditionally. React Hooks must be called in the exact same order in every component render.
This is because you used the hook after some conditional rendering.

Reply:
Maciej specified the colors [here](https://github.com/elastic/actionable-observability/issues/124#issuecomment-1731271446). I fixed the `useEuiBackgroundColor` conditional rendering problem